### PR TITLE
Create ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59816](https://cumulocity.atlassian.net/browse/MTM-59816)
Original PR: [6656](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6656)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: adjust widget title and actions for overlay and hidden appearance (#6643) [GRAFT][release/cd] (#6656)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Widget title and actions adjusted for overlay and hidden appearance
```

File: content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59816
+version: 1020.2.13
+---
+adjust widget title and actions for overlay and hidden appearance (#6643) [GRAFT][release/cd] (#6656)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve the user experience when configuring widgets, the widget title and action buttons have been adjusted for the overlay and hidden widget appearance settings. Previously, the title and action buttons were always shown which could be confusing in some configurations. With this change, the title and action buttons are now hidden when either the overlay or hidden appearance is selected in the widget settings. This impacts the widget configuration for all users, but the actual widget appearance and behavior for end users remains unchanged.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by carlosceia — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59816
+version: 1020.2.13
+---
+The display of widget titles and action buttons has been optimized to align with the overlay and hidden widget appearance settings. Previously, these elements were always visible, which could be confusing when edit widgets mode was disabled. With this update, the action button is now only displayed when either the overlay or hidden appearance is selected in the widget settings and edit widgets mode is enabled.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md:17]:
```suggestion
The display of widget titles and action buttons has been optimized to align with the overlay and hidden widget appearance settings. Previously, these elements were always visible, which could be confusing when the edit widgets mode was disabled. With this update, the action button is now only displayed when either the overlay or hidden appearance is selected in the widget settings and the edit widgets mode is enabled. 
```

File: content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Widget title and actions adjusted for overlay and hidden appearance
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md:3]:
```suggestion
title: Widget title and actions aligned with overlay and hidden appearance settings
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**